### PR TITLE
Lenovo doc update

### PR DESCRIPTION
**Pull request by amuraleedhar**
##### SUMMARY
In this PR I am adding documentation for ENOS and CNOS.  platform_enos.rst and platform_cnos.rst are added that will come as hyperlink to those entries in index

##### ISSUE TYPE
 - Documentation Pull Request

##### COMPONENT NAME

docs/docsite/rst/network/user_guide/platform_index.rst
docs/docsite/rst/network/user_guide/platform_cnos.rst
docs/docsite/rst/network/user_guide/platform_enos.rst

##### ANSIBLE VERSION
ansible 2.7.0.dev0 (devel f9cbdcd) last updated 2018/07/03 14:55:43 (GMT +550)
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /home/ansible/sheru/ansible/lib/ansible
executable location = /home/ansible/sheru/ansible/bin/ansible
python version = 2.7.6 (default, Nov 23 2017, 15:49:48) [GCC 4.8.4]

##### ADDITIONAL INFORMATION
None.

### CODE REVIEW

**Comment by samccann** [comment]
Thanks for updating the documentation!  I'll take a look and get back to you if I find any issues.

File: docs/docsite/rst/network/user_guide/platform_enos.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. _enos_platform_options:
+
+***************************************
+ENOS Platform Options
+***************************************
+
+ENOS supports Enable Mode (Privilege Escalation). This page offers details on how to use Enable Mode on ENOS in Ansible 2.7.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_enos.rst]: Same issue - if others agree, remove 2.7 here

File: docs/docsite/rst/network/user_guide/platform_enos.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      |
+| |                         | | and ``ansible_become_pass:``                |
++---------------------------+-----------------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                |
++---------------------------+-----------------------------------------------+
+
+For legacy playbooks, ENOS still supports ``ansible_connection: local``. We recommend modernizing to use ``ansible_connection: network_cli`` as soon as possible.
+
+Using CLI in Ansible 2.7
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_enos.rst]: Same comment - if we can remove versions in this section, then remove '2.7' here as well.

File: docs/docsite/rst/network/user_guide/platform_cnos.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.. _cnos_platform_options:
+
+***************************************
+CNOS Platform Options
+***************************************
+
+CNOS supports Enable Mode (Privilege Escalation). This page offers details on how to use Enable Mode on CNOS in Ansible 2.7.
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_cnos.rst]: @gundalow @privateip Is there any reason we should keep a specific version in these platform guides?
@amuraleedhar This version says 2.7 but the settings by platform table entries say this is supported as of 2.5. Let's see what the other two folks say, but I think we can remove the specific version in this intro sentence.

File: docs/docsite/rst/network/user_guide/platform_cnos.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      |
+| |                         | | and ``ansible_become_pass:``                |
++---------------------------+-----------------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                |
++---------------------------+-----------------------------------------------+
+
+For legacy playbooks, CNOS still supports ``ansible_connection: local``. We recommend modernizing to use ``ansible_connection: network_cli`` as soon as possible.
+
+Using CLI in Ansible 2.7
```
Comment (samccann) [inline docs/docsite/rst/network/user_guide/platform_cnos.rst]: Same issue, if the others agree, remove 2.7 here.

File: docs/docsite/rst/network/user_guide/platform_cnos.rst (hunk 3 above)
Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: Removing 2.7 now. Let it be generic. For specific reasons users may refer to the table.

File: docs/docsite/rst/network/user_guide/platform_cnos.rst (hunk 4 above)
Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: All other vendors say ios, eos, junos all are putting the version 2.6 there. So I too followed the same. I feel it make sense to remove version from there. Whether to use network_cli or local is the confusion point for user, then he may refer the table in the platform_index.rst. I am waiting for Gundalow and/or privateip to respond.

Comment (samccann) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: Yes, I'm thinking I need to go back in and remove 2.6 from all the others as well. But let's see if @gundalow, @privateip agree.

Comment (samccann) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: I chatted with @gundalow in slack and he is okay with removing the version. I will go back and remove the 2.6 version from the other guides in https://github.com/ansible/ansible/issues/45630

Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: I will remove 2.7 from every where now. 
Thanks 
-Anil

File: docs/docsite/rst/network/user_guide/platform_cnos.rst (hunk 3 above)
Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_cnos.rst]: Removing 2.7 now

File: docs/docsite/rst/network/user_guide/platform_enos.rst (hunk 1 above)
Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_enos.rst]: Removing 2.7

File: docs/docsite/rst/network/user_guide/platform_enos.rst (hunk 2 above)
Comment (amuraleedhar) [reply docs/docsite/rst/network/user_guide/platform_enos.rst]: Done

**Review by samccann — APPROVED** [review]

**Pull request merged**